SportsNews/Migrations/20220531094946_AddModels02.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace SportsNews.Migrations
+{
+    public partial class AddModels02 : Migration
```

Comment:
Change Migration Name https://stackoverflow.com/questions/32480267/is-there-a-best-or-recommended-practice-for-naming-entity-framework-migrations